modules/doc/content/getting_started/learning_resources/index.md

```diff
@@ -0,0 +1,8 @@
+# Learning Resources
+
+This following pages provide some fundamentals that may be needed to use or
+develop MOOSE and/or MOOSE-based applications:
+
+- [Finite Element Method Fundamentals](finite_element_concepts/index.md)
+- [C++ Fundamentals](help/c++/index.md)
+- [Git Fundamentals](git.md)
```

Comment:
I d point to Application Development or some of the sub-catageories of App dev

Reply:
I meant for this page to just list out the not-MOOSE links that are largely outside of our scope

Reply:
maybe call it additional learning resources?